detectron2/utils/visualizer.py

```diff
@@ -726,6 +726,7 @@ def overlay_instances(
 
                 height_ratio = (y1 - y0) / np.sqrt(self.output.height * self.output.width)
                 lighter_color = self._change_color_brightness(color, brightness_factor=0.7)
+                lighter_color = tuple(np.clip(lighter_color, 0.0, 1.0))
```

Comment:
Thanks, could you add it into _change_color_brightness instead?

Reply:
Done. Also signed CLA